lib/constants.js

```diff
@@ -22,3 +22,5 @@ export const DONT_LIKE_THIS_COST = 1
 // to be loaded from the server
 export const SUBS = ['bitcoin', 'nostr', 'tech', 'meta', 'jobs']
 export const SUBS_NO_JOBS = SUBS.filter(s => s !== 'jobs')
+
+export const ANON_USER_NAME = 'anon'
```

Comment:
we should replace this and all corresponding logic with the anon user id after the user was created in production